### PR TITLE
Fix Function NPE

### PR DESCRIPTION
**Pull request by UnderscoreTud**
### Description
This PR fixes an NPE which results when trying to use functions in an effect command

---
**Target Minecraft Versions:** any
**Requirements:** none
**Related Issues:** none

### CODE REVIEW

**Review by TheLimeGlass — APPROVED** [review]

File: src/main/java/ch/njol/skript/lang/function/Namespace.java [hunk 1]
```diff
@@ -50,46 +51,44 @@ public enum Origin {
...
 		
 		public Origin getOrigin() {
 			return origin;
 		}
-		
-		public String getName() {
-			return name;
+
+		@Nullable
+		public String getScript() {
```
Comment (APickledWalrus) [inline src/main/java/ch/njol/skript/lang/function/Namespace.java:69]: getScriptName maybe? Also is the old method not used anywhere?

**Review by APickledWalrus — APPROVED** [review]

File: src/main/java/ch/njol/skript/lang/function/Namespace.java (hunk 1 above)
Comment (UnderscoreTud) [reply src/main/java/ch/njol/skript/lang/function/Namespace.java:69]: Oh right, I always forget about backwards compatibility 😭 I'll revert the name changes

Comment (TheLimeGlass) [reply src/main/java/ch/njol/skript/lang/function/Namespace.java:69]: It wasn't used, atleast by Skript.

**Review by Moderocky — APPROVED** [review]

File: src/main/java/ch/njol/skript/lang/function/Namespace.java [hunk 2]
```diff
@@ -50,46 +51,44 @@ public enum Origin {
...
 		}
 
 		@Override
 		public int hashCode() {
-			int prime = 31;
-			int result = 1;
-			result = prime * result + name.hashCode();
-			result = prime * result + origin.hashCode();
+			int result = origin.hashCode();
+			result = 31 * result + (script != null ? script.hashCode() : 0);
```
[the remark this answers is not in the document]
Comment (UnderscoreTud) [reply src/main/java/ch/njol/skript/lang/function/Namespace.java:76]: it's generated using intellij

**Review by AyhamAl-Ali — APPROVED** [review]
Looking good!

File: src/main/java/ch/njol/skript/lang/function/Namespace.java (hunk 1 above)
Comment (TheLimeGlass) [inline src/main/java/ch/njol/skript/lang/function/Namespace.java:69]: I agree 
```suggestion
		public String getScriptName() {
```

**Review by TheLimeGlass — CHANGES_REQUESTED** [review]

**Review by TheLimeGlass — APPROVED** [review]

**Pull request merged**